lib/geordi/gitlinear.rb

```diff
@@ -0,0 +1,201 @@
+require 'highline'
+require 'net/http'
+require 'json'
+require 'active_support/core_ext/object/blank'
```

Comment:
I think we cannot rely on active_support being around. Can you make it without?

Reply:
I think you don't even use it :)

Reply:
`require 'active_support/core_ext/object/blank'` was needed for `if branch_name.present?` in L49, as `.present?` is defined in it. I replaced `.present?` with `.nil?`. IMO this should also be sufficient since `.find` returns `nil` by default if no object matches. Can you please take a quick look at it to make sure that's also okay here?

Reply:
I think you could write `branch_list_string.to_s.strip.empty?` to cover all cases (no `.nil?` check needed).